web/src/main/resources/static/src/js/controller/GenomeNexus.js

```diff
@@ -62,17 +62,6 @@ function GenomeNexus(options)
             },
             home: {
                 pageInfo: 'Genome Nexus, a comprehensive one-stop resource for fast, automated and high-throughput annotation and interpretation of genetic variants in cancer. Genome Nexus will integrate information from a variety of existing resources, including databases that convert DNA changes to protein changes, predict the functional effects of protein mutations, and contain information about mutation frequencies, gene function, variant effects, and clinical actionability.<br/><br/>Three goals:<br/>1. Data collection from various annotation sources<br/>2. Integration of heterogeneous information into a harmonized structure and programmatic interface<br/>3. Dissemination of the diverse information in a hierarchical digestible way for interpreting variants and patients.<br/><br/>Genome Nexus is an open source software project. You can find the source code in our <a href="https://github.com/cBioPortal/genome-nexus" style="color:blue">github repository</a> and documentation for our web service <a href="./swagger-ui.html" style="color:blue">here</a>.<br/><br/><img src="lib/images/gn_figure.png" alt="Genome Nexus" width="800">' // _.template($("#default_mutation_info").html())()
-            },
-            download: {
-                content:[{
-                    links: [
-                        {href: 'href="files/hotspots.xls"',
```

Comment:
Trying to make it less obvious that we copied & pasted everything from cancer hotspots repo :)